.github/workflows/windows-build.yml

```diff
@@ -194,8 +208,8 @@ jobs:
           try {
             tar --format=posix -cvpf ${{ env.DEV_DRIVE_WORKSPACE }}\cache.tar `
               .ccache `
-              .o3de\python `
-              3rdParty\packages `
+              .o3de\python\downloaded_packages `
```

Comment:
Note: Once this is merged in, the first profile build will have a non-blocking compress artifacts error, as the last artifact it pulls in does not have any files from downloaded_packages